### PR TITLE
Enable clang-tidy on torch/csrc/distributed/rpc

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: .lintrunner.toml [hunk 1]
```diff
@@ -216,8 +216,10 @@ include_patterns = [
     'torch/csrc/*.cpp',
     'torch/csrc/**/*.h',
     'torch/csrc/**/*.cpp',
-    'torch/csrc/distributed/autograd/*.cpp',
-    'torch/csrc/distributed/autograd/*.h',
+    'torch/csrc/distributed/autograd/**/*.cpp',
```
Comment (Skylion007) [inline .lintrunner.toml:219]: This is a no-op and should be covered by lines 217-218. Is there a bug with lintrunner that needs to be patched?

Comment (cyyever) [reply .lintrunner.toml:219]: torch/csrc/distributed was excluded in the excluded pattern. I want to gradually enable it by specifying sub-directories.

**Review by Skylion007 — APPROVED** [review]

**Pull request closed without merging**